script/01-gaussian-mixture-model/01_gaussian-mixture-model.jl

```diff
@@ -23,46 +23,56 @@ using Turing, MCMCChains
 Turing.setprogress!(false);
 
 
-@model GaussianMixtureModel(x) = begin
-    
+@model function GaussianMixtureModel(x)
+
     D, N = size(x)
 
     # Draw the parameters for cluster 1.
     μ1 ~ Normal()
-    
+
     # Draw the parameters for cluster 2.
     μ2 ~ Normal()
-    
+
     μ = [μ1, μ2]
-    
-    # Uncomment the following lines to draw the weights for the K clusters 
+
+    # Uncomment the following lines to draw the weights for the K clusters
     # from a Dirichlet distribution.
-    
+
     # α = 1.0
-    # w ~ Dirichlet(2, α)
-    
+    # w ~ Dirichlet(2, α)
+
     # Comment out this line if you instead want to draw the weights.
     w = [0.5, 0.5]
-    
+
     # Draw assignments for each datum and generate it from a multivariate normal.
     k = Vector{Int}(undef, N)
     for i in 1:N
         k[i] ~ Categorical(w)
-        x[:,i] ~ MvNormal([μ[k[i]], μ[k[i]]], 1.)
+        x[:,i] ~ MvNormal([μ[k[i]], μ[k[i]]], 1.)
```

Comment:
This MvNormal constructor is deprecated.

Reply:
Aha. I see. I also see this problem in tutorial:

- 03 Bayesian neural network
- 05 Linear regression
- 08 Multinomial logistic regression
- 09 Variational inference
- 10 Bayesian differential equations

I'll open a separate issue for this.